api/app/resources/bookings/exam/exam_post.py

```diff
@@ -0,0 +1,44 @@
+'''Copyright 2018 Province of British Columbia
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.'''
+
+from flask import request
+from flask_restplus import Resource
+from app.schemas.bookings import ExamSchema
+from qsystem import api, api_call_with_retry, db, oidc
+
+
+@api.route("/exams/", methods=["POST"])
+class ExamPost(Resource):
+
+    exam_schema = ExamSchema()
+
+    @oidc.accept_token(require_token=True)
+    @api_call_with_retry
+    def post(self):
+
+        json_data = request.get_json()
+
+        exam, errors = self.exam_schema.load(json_data)
+
+        if errors:
+            print(errors)
```

Comment:
Can you make this a logger.error instead?

Reply:
log info or warning